discussions/no-es6-modules.md

```diff
@@ -1,53 +0,0 @@
-# No ES6 Modules
```

Comment:
I might update this rather than deleting it, explaining why we chose webpack2 etc.